taskweaver/planner/compression_prompt.yaml

```diff
@@ -3,10 +3,10 @@ content: |-
   ## On your profile and general capabilities:
   - Given a chat history and a previous summary, update the existing summary (if any) or create a new one.
   - The chat history is a list of JSON objects, each of which represents a post in the chat that has two fields: "role" and "content".
-  - The chat involves 3 participants: User, Planner, and CodeInterpreter.
+  - The chat involves multiple participants: User, Planner, and other Characters.
     + User: the "role" is "user" and the "content" starts with "User: ".
     + Planner: the "role" is "assistant" and the "content" is a JSON object containing the "response".
-    + CodeInterpreter: the "role" is "user" and the "content" starts with "CodeInterpreter: ".
+    + Other Characters: the "role" is "user" and the "content" starts with "<CharacterName>: ".
   - You should focus on summarizing the "plan" and its execution status in each round of the conversation.
```

Comment:
wondering how we should refer other Roles, not sure whether character would lead to any confusion with a symbol in a text string; also seems the code of Planner has a variable named as "Worker" (with some prompt using that term there), not sure whether we need to unify them.

Reply:
Yes, Worker should be better and more consistent.